src/SFA.DAS.PSRService.Web/Controllers/ReportController.cs

```diff
@@ -267,6 +295,53 @@ public IActionResult OrganisationName(string post)
             return View("OrganisationName", organisationVm);
         }
 
+        [Route("TotalEmployees")]
+        [Authorize(Policy = PolicyNames.CanEditReport)]
+        public IActionResult TotalEmployees()
+        {
+            var report = _reportService.GetReport(_currentPeriod.PeriodString, EmployerAccount.AccountId);
+
+            if (!_reportService.CanBeEdited(report))
+                return new RedirectResult(Url.Action("Index", "Home"));
+
+            var totalEmployeesVm = new TotalEmployees
+            {
+                HasTotalEmployeesMeetMinimum = report.HasTotalEmployeesMeetMinimum,
+                Report = report
+            };
+
+            ViewBag.CurrentPeriod = report?.Period ?? _currentPeriod;
+
+            return View("TotalEmployees", totalEmployeesVm);
+        }
+
+        [Route("PostTotalEmployees")]
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        [Authorize(Policy = PolicyNames.CanEditReport)]
+        public IActionResult PostTotalEmployees(TotalEmployees totalEmployeesVm)
```

Comment:
why are we posting this ViewModel? It makes no sense. The report property isn't populated. We populate in this action method. Then we take that property and pass it along essentially throwing away the viewModel object. The post the `HasTotalEmployeesMeetMinimum` as a param. Also think about renaming `HasTotalEmployeesMeetMinimum`. `HasMinimumEmployeeHeadcount` would be better